ludwig/trainers/trainer_llm.py

```diff
@@ -503,37 +503,53 @@ def _create_batch_size_evaluator(self) -> BatchSizeEvaluator:
         trainer = self
 
         class _TrainerBatchSizeEvaluator(BatchSizeEvaluator):
+            def __init__(self):
+                self.input_feature_name, self.input_feature = trainer.model.input_features.items()[0]
+                self.output_feature_name, self.output_feature = trainer.model.output_features.items()[0]
+
+                # Get the length of the longest input sequence from the training data
+                self.input_msl = self.input_feature.input_shape[0]
+                # Get the length of the longest output sequence from the training data
+                self.output_msl = self.output_feature.output_shape[0]
```

Comment:
I don't know if this actually gets the longest output sequence. I checked the code for output_shape, and I saw that it actually generated outputs from the input sequence and didn't seem to use the targets provided.

Reply:
nvm scratch this